spinnaker-monitoring-daemon/spinnaker-monitoring/datadog_service.py

```diff
@@ -146,8 +148,16 @@ def read_param(param_name, config_object):
   if host is None and datadog_host is None:
     host = ''
   host = socket.getfqdn(host) if host is not None else datadog_host
-
-  return DatadogMetricsService(api_key=api_key, app_key=app_key, host=host)
+  static_datadog_tags = []
+  for tag_pair in filter(None, os.getenv('DATADOG_TAGS', '').replace(' ', '').split(',')):
+    if ':' in tag_pair:
+      k, v = tag_pair.split(':')
```

Comment:
Why ':' instead of '=' ? Is this a datadog-ism?

Reply:
yep.  Datadog tags look like: https://github.com/DataDog/dd-agent/blob/master/datadog.conf.example#L36-L37